Merge Implementing Matomo blog post from dev branch (#232)
RockefellerArchiveCenter/bits_and_bytes: _posts/2023-10-05-implementing-matomo.md

--- 
title: " Implementing Matomo for Web Analytics"
date: 2023-10-04T11:10:00
author: 
- Andrea Cadornigara
- Erich Chang
- Kathleen Leonard
- Hannah Sistrunk
- Darren Young
layout: post
categories:
  - Collaboration
  - Software and Systems
tags:
  - web analytics
  - learning
excerpt_separator: <!--more--> 
---

> We are the Web Analytics Owners Group, an internal Rockefeller Archive Center (RAC) working group formed with 5 staff members from each of our program areas. As we prepared for the scheduled sunsetting of Google Analytics in July 2023, our team undertook a project in December 2021 to find an alternative analytics platform that fit our needs. We [analyzed and compared 5 different platforms](https://lucid.app/users/registerOrLogin/free?showLogin=false&invitationId=inv_8975cd1a-a6df-471b-b6f2-402f641b1294&productOpt=chart&invitationType=documentAcceptance&returnUrlOverride=%2Flucidchart%2F23f2ce1d-f1cd-4ba8-8e64-d14d1868e68e%2Fedit%3Fviewport_loc%3D276%252C-124%252C2107%252C1065%252C0_0%26invitationId%3Dinv_8975cd1a-a6df-471b-b6f2-402f641b1294) and ultimately chose Matomo because it was user-friendly, aligned with our requirements, and emphasizes user privacy. In this post, we’ll detail how we implemented Matomo at the RAC and what the next steps are for our group.

Reviewer: ```suggestion
We are the Web Analytics Owners Group, an internal Rockefeller Archive Center (RAC) working group formed with 5 staff members from each of our program areas. As we prepared for the scheduled sunsetting of Google Analytics in July 2023, our team undertook a project in December 2021 to find an alternative analytics platform that fit our needs. We [analyzed and compared 5 different platforms](https://blog.rockarch.org/moving-from-google-matomo-to-put-the-we-in-web-analytics) and ultimately chose Matomo because it was user-friendly, aligned with our requirements, and emphasizes user privacy. In this post, we’ll detail how we implemented Matomo at the RAC and what the next steps are for our group.

```

4. **Update the code** on [each of our websites](https://github.com/RockefellerArchiveCenter) to start collecting Matomo analytics data.
5. **Implement an account provisioning process** for Rockefeller Archive Center staff who need access to Matomo. Once Matomo was configured and gathering data, we worked with our IT colleagues to get RAC staff the access they required.
6. **Execute a training program** for staff to understand the basics of how to use the tool and what kinds of questions can be answered using analytics. We planned and conducted training for our leadership team, and then each Web Analytics Owners Group member took the training back to their team to train our colleagues as needed.

## Organizing Our Implementation Processes

> We outlined three distinct phases of work in our project charter for implementing Matomo: research, implementation, and training. After we completed these three phases, we also set aside time for reflection on the project and strategizing our group’s future directions.

Reviewer: ```suggestion
We outlined three distinct phases of work in our project plan for implementing Matomo: research, implementation, and training. After we completed these three phases, we also set aside time for reflection on the project and strategizing our group’s future directions.

```

--- 

> title: " Implementing Matomo for Web Analytics"

Reviewer 2: ```suggestion
title: "Implementing Matomo for Web Analytics"

```

Reviewer 2: delete space after quotation mark
